core/server/api_container/server/startosis_engine/kurtosis_types/service_config/image_load_spec.go

```diff
@@ -0,0 +1,136 @@
+package service_config
+
+import (
+	"path"
+
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/image_load"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_starlark_framework"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/builtin_argument"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/kurtosis_type_constructor"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/startosis_constants"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/startosis_errors"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/startosis_packages"
+	"go.starlark.net/starlark"
+)
+
+const (
+	ImageLoadTypeName = "ImageLoad"
+	LoadImageAttr     = "image_file_path"
+)
+
+func NewImageLoadType() *kurtosis_type_constructor.KurtosisTypeConstructor {
```

Comment:
Same question as above - makes a lot of sense to me that a KurtosisBackend needs the ability to load images, but do we expect users to be writing Starlark that references an image .tgz?